nursery/execute-syscall.yml

```diff
@@ -1,19 +1,22 @@
 rule:
   meta:
-    name: execute syscall instruction
+    name: execute syscall
     namespace: anti-analysis
     authors:
       - "@kulinacs"
       - "@mr-tz"
+      - mehunhoff@google.com
     description: may be used to evade hooks or hinder analysis
     scopes:
       static: basic block
       dynamic: unsupported  # requires mnemonic features
     references:
       - https://github.com/j00ru/windows-syscalls
   features:
-    - and:
-      - mnemonic: syscall
-      - or:
-        - mnemonic: ret
-        - mnemonic: retn
+    - or:
+      - api: syscall
```

Comment:
can you add a comment for this? what function/runtime?

Reply:
It's https://man7.org/linux/man-pages/man2/syscall.2.html. I'm not sure what you're asking here. I can add `os` (`linux`, `android`)?

Reply:
sorry I wasn't more clear. 

Yeah I wasn't sure if it was some GCC built in, or a linux thing, or something from another platform, etc. So, I think a comment or description would help. Also constraining the OS makes sense too, since it's Linux specific.